usagov/usagov-2021
# Usagov 380 false children

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-380

## Description

Removes Related Pages field now that false children can be managed in the Menu structure.

## How To Test
 - Go to Structure > Menus > Left Menu English > Add Link
 - Fill out the fields to add a page and set its parent to be one of the Navigation Cards pages
   - [x] Verify that the added page is displayed on the nav page in the same order as the menu
 - Drag and Drop the added menu item to change the order
   - [x] Verify that the added page is displayed on the nav page in the same order as the menu
 - Change the page to a Navigation page (without cards)
   - [x] Verify that the order is the same in the left-nav
 - Change the page into a Life Events page
   - [x] Verify that the added page is displayed properly here as well

## Checklist for the Developer



## Checklist for the Peer Reviewers
